pkg/option/resolver/resolver.go

```diff
@@ -230,12 +231,54 @@ func readConfigMap(ctx context.Context, client client.Clientset, source ConfigSo
 	return cm.Data, []string{source.String()}, nil
 }
 
-// ReadConfigOverrides reads all the CiliumNodeConfig objects and returns a flattened map
+// readNodeConfigsAllVersions read node configurations for versions v2 and v2alpha1 of CiliumNodeConfig CRD.
+// TODO remove me when CiliumNodeConfig v2alpha1 is depreacted
+func readNodeConfigsAllVersions(ctx context.Context, client client.Clientset, nodeName, namespace, name string) (map[string]string, []string, error) {
```

Comment:
[1]

Reply:
I opened the following issue https://github.com/cilium/cilium/issues/31982